templates/tfengine/components/devops/main.tf

```diff
@@ -50,36 +50,30 @@ module "project" {
   source  = "terraform-google-modules/project-factory/google"
   version = "~> 10.2.2"
 
-  name            = "{{.project.project_id}}"
+  name            = var.project.project_id
   {{- if eq .parent_type "organization"}}
-  org_id          = "{{.parent_id}}"
+  org_id          = var.parent_id
   {{- else}}
   org_id          = ""
-  folder_id       = "{{.parent_id}}"
+  folder_id       = var.parent_id
   {{- end}}
-  billing_account = "{{.billing_account}}"
+  billing_account = var.billing_account
   lien            = {{get . "enable_lien" true}}
   # Create and keep default service accounts when certain APIs are enabled.
   default_service_account = "keep"
   # Do not create an additional project service account to be used for Compute Engine.
   create_project_sa = false
-  activate_apis = [
-    "cloudbuild.googleapis.com",
-    "cloudidentity.googleapis.com",
-    {{range get . "project.apis" -}}
-    "{{.}}",
-    {{end -}}
-  ]
+  activate_apis = var.project.apis
```

Comment:
can you do `var.project.apis`? like an extra dot in the var?

Reply:
Yeah, it works. Only crashes when `.project` is null, so we have to be careful to only let them appear when the object containing is required, otherwise should be wrapped on an `if` block to check if the container object exists

Reply:
Resolving this.

Conclusion: It works but is needed to validate nested field against private catalog. Left a `TODO` as requested at https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/908#discussion_r660085280